content/800-data-platform/050-data-proxy/700-prisma-cli-with-data-proxy.mdx

```diff
@@ -35,29 +25,42 @@ If your database is behind a firewall, add your local IP address to the allowlis
 - `prisma db execute`
 - `prisma studio`
 
-### Solution
+In Prisma versions 4.9.0 and later, you can [set a direct database connection URL in your Prisma schema](#set-a-direct-database-connection-url-in-your-prisma-schema).
 
-Assuming that you have `env("DATABASE_URL")` in your `schema.prisma` file, you must override the `DATABASE_URL` environment variable before you run `npx prisma migrate deploy` or any of the commands in the list above.
+In earlier versions, you must add an [environment variable](/guides/development-environment/environment-variables) with the direct database connection URL and overwrite the [`DATABASE_URL` environment variable](/guides/development-environment/environment-variables#example-set-the-database_url-environment-variable-in-an-env-file) when you need a direct connection.
 
-You can declare the environment variable `MIGRATE_DATABASE_URL`, and use that to override the connection.
+<Admonition type="warning">
 
-For example:
+**Important**<br /><br />
 
-```bash file=.env highlight=2;add
-DATABASE_URL="prisma://..."
-MIGRATE_DATABASE_URL="postgresql://..."
+If your database is behind a firewall, add your local IP address to the allowlist of your database.
 
-```
+</Admonition>
```

Comment:
This feels a bit overly cautious here. This is true for all connection strings of Prisma - and we do not always mention that. Not sure if this is more relevant for Data Proxy users.

Reply:
Yes, this was already in there and just got moved... wasn't sure whether to keep it. Any thoughts @zachtil?

Reply:
Well, I remember struggling to figure out why I could not connect from my local environment to MongoDB Atlas and that prompted me to add this note here as I was anticipating other cloud databases might have the same issues.

From another viewpoint, this is a general note and not a Prisma-specific one. We could easily do without it.

Reply:
I am ok to remove it. Jan's comment makes sense.

Reply:
Thanks, removed now